app/Http/Controllers/Store/RegistrationController.php

```diff
@@ -448,14 +449,11 @@ function ($new_carer) {
             (array)$request->get('children')
         );
 
-        // Grab the date
-        $now = Carbon::now();
```

Comment:
Assuming `$now` and `$user` were unused from before and this is a tidy up?

Reply:
yep, IDE highlighted them as unused.